api/cas-server-core-api-configuration/src/main/java/org/apereo/cas/configuration/support/CasConfigurationJasyptCipherExecutor.java

```diff
@@ -179,24 +191,37 @@ public String encryptValue(final String value) {
      */
     public String decryptValue(final String value) {
         try {
-            if (StringUtils.isNotBlank(value) && value.startsWith(ENCRYPTED_VALUE_PREFIX)) {
-                initializeJasyptInstanceIfNecessary();
-                final String encValue = value.substring(ENCRYPTED_VALUE_PREFIX.length());
-                LOGGER.trace("Decrypting value [{}]...", encValue);
-                final String result = this.jasyptInstance.decrypt(encValue);
-                if (StringUtils.isNotBlank(result)) {
-                    LOGGER.debug("Decrypted value [{}] successfully.", encValue);
-                    return result;
-                }
-                LOGGER.warn("Encrypted value [{}] has no values.", encValue);
-            }
-            return value;
+            return decryptValue2(value);
         } catch (final Exception e) {
             LOGGER.error("Could not decrypt value [{}]", e);
         }
         return null;
     }
 
+    /**
+     * Decrypt value string. (but don't log error, for use in shell).
+     *
+     * @param value the value
+     * @return the string
+     */
+    public String decryptValue2(final String value) {
```

Comment:
Please rename these methods, removing the number 2, into something that is a bit more intention-revealing.